docs/USAGE-GUIDE.md

```diff
@@ -1197,6 +1197,53 @@ int s2n_early_data_cb_async_impl(struct s2n_connection *conn, struct s2n_offered
 
 **s2n_offered_early_data_get_context_length** and **s2n_offered_early_data_get_context** can be called to examine the optional user context associated with the early data. Unlike most s2n-tls callbacks, the context is not configured when the callback is set. Instead, the context is associated with the specific pre-shared key or session ticket used for early data. The context can be set for external pre-shared keys by calling **s2n_psk_set_early_data_context**. For session tickets, **s2n_connection_set_server_early_data_context** can be used to set the context the server includes on its new session tickets. Because the server needs to serialize the context when creating a new session ticket, the context is a byte buffer instead of the usual void pointer.
 
+# Callbacks
+
+The following is a list of all callbacks types and functions available for s2n-tls. Callbacks supported in the Rust bindings are marked with `[Rust]`.
```

Comment:
Does this make sense to have here? Or should we have a separate Rust guide?

Reply:
As Lindsay mentioned, callbacks are not special and simply APIs so it doesnt make sense to have a separate section for them. I drafted the section for my own benefit but I dont think its needed for general users.